demos/calibration/calibration-pubmed-link-prediction.ipynb

```diff
@@ -0,0 +1,1795 @@
+{
```

Comment:
<p>You mention that there is no dropout, but currently it is set to 0.2</p>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>